### ISSUE TITLE
v1.0 of nmfs-opensci contributor guide

### ISSUE THREAD

**Issue by eeholmes** [nmfs-opensci/Contributor-Guide#1]
@k-doering-NOAA @jmlondon First stab at a very simple and to the point GH org contributor guide. https://nmfs-opensci.github.io/Contributor-Guide/. Idea is to edit this and to have it (or a copy) be a template that other GH org admin can use as a simple template for their orgs.

To do

- [ ] Edit and improve content
- [ ] Decide on a better set of sections
- [ ] Add links to GGT content when we get that done
- [ ] Add links and instructions for using template for creating repos with the basic content (disclaimers, license, coc)

**Comment by k-doering-NOAA** [comment]
Good idea! One thing I noticed is the contributor covenant is an older version - is there a good reason for that? Here's an example Christine and I put together to use for FIT and FIMS (we also added a reporting form to allow reporting anonymously - thanks to openscapes for giving us the space to do this! :) ) https://github.com/nmfs-fish-tools/Resources/blob/main/CODE_OF_CONDUCT.md

**Comment by k-doering-NOAA** [comment]
> The nmfs-opensci GitHub organization is not intended for official NOAA communications and NOAA logos should not generally be used on repositories.

I was honestly surprised by this! nmfs-opensci seems like it should be as "NOAA branded" as the FIT is...  maybe a long-form discussion for us to have sometime? :)

**Comment by eeholmes** [comment]
@k-doering-NOAA Thanks for catching the old CoC. I updated to the newest one. 

Re

> The nmfs-opensci GitHub organization is not intended for official NOAA communications and NOAA logos should not generally be used on repositories.

GitHub use is covered under the Social Media policy per the last GitHub policy, and use of logos requires applying to use those, which I have not done. https://www.commerce.gov/about/policies/social-media.  So I didn't want us to run afoul of that.  My take-home from the social media policy is that one needs to be careful not to present content on social media as an official communication from NOAA.

Also technically NMFS Open Science has not been announced yet, so we are not a NOAA initiative yet. I need to work with my OST contact about getting a webpage up on fisheries.noaa.gov, which will make it feel more official.

**Comment by k-doering-NOAA** [comment]
ok, good to know!

What other feedback would be useful for you? Editing? Or just broader comments, such as about the Code of Conduct?

**Comment by eeholmes** [comment]
The following would be good feedback. Viewing this as a "template" of a minimal GH org Contributor Guide:

* what tabs are needed? The ones I have now are a first start.
* should content be added or moved around?
